appcelerator/titanium_mobile
# [TIMOB-24579] Android: recognise newer attributes as valid tokens

**JIRA:** https://jira.appcelerator.org/browse/TIMOB-24579

#### Test case:
1. Add attribute
2. Run app
3. Check if attribute is in build/android/AndroidManifest.xml. If it is present then test has passed.
